packages/core/lib/commands/compile/run.js

```diff
@@ -23,6 +23,33 @@ module.exports = async function (options) {
     );
   }
 
+  if (config._ && config._.length > 0) {
```

Comment:
what is `config._`?

Reply:
`_` is an array where arguments from the command line get put. In this case it will be an array populated by source paths.

Reply:
thanks!